browser/resources/brave_new_tab_page_refresh/components/background/background.tsx

```diff
@@ -176,7 +176,7 @@ function IframeBackground(props: IframeBackgroundProps) {
   }, [props.expectedOrigin, props.onMessage])
 
   React.useEffect(() => {
-    if (!props.onReady) {
+    if (!props.onReady || !contentLoaded) {
```

Comment:
I think `props.onReady` should never be falsey, as its just the setter for React.useState - could we update the type to reflect that?